### PR TITLE
RELATED: ONE-4621 - Improve code documentation for attribute element types

### PR DESCRIPTION
**Pull request by lupko**
---

Supported PR commands:

| Command         | Description            |
| --------------- | ---------------------- |
| `ok to test`    | Re-run standard checks |
| `extended test` | BackstopJS tests       |

---

# PR Checklist

-   [ ] commit messages adhere to the [commit message guidelines](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#what-should-the-commits-look-like)
-   [ ] `check` passes
-   [ ] `check-extended` passes
-   [ ] `rush change` [was run if applicable](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-describe-my-changes-for-the-changelog)

### CODE REVIEW

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/677/ : SUCCESS in 3s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/2264/ : FAILURE in 1m 24s

File: libs/sdk-model/src/execution/filter/index.ts [hunk 1]
```diff
@@ -53,6 +56,13 @@ export interface IPositiveAttributeFilter {
          * Display form whose attribute elements are included in the 'in' list.
          */
         displayForm: ObjRef;
+
+        /**
+         * Attribute elements to filter in. The attribute elements can be specified either using
+         * their human readable value or by using they URI = the primary key. Using either representation has
```
Comment (no23reason) [inline libs/sdk-model/src/execution/filter/index.ts:62]: Typo: their primary

File: libs/sdk-backend-spi/src/workspace/execution/results.ts [hunk 2]
```diff
@@ -202,8 +202,26 @@ export interface IDimensionDescriptor {
...
     attributeHeaderItem: {
-        uri: string;
+        /**
+         * Human readable name of the attribute element
+         */
         name: string;
+
+        /**
+         * URI of the attribute element. This is essentially a primary key of the attribute element. It can
+         * be used in places where attribute elements have to be exactly specifies - such as positive or
```
Comment (no23reason) [inline libs/sdk-backend-spi/src/workspace/execution/results.ts:212]: Typo: specified

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Review by no23reason — APPROVED** [review]
LGTM

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul-el7/307/ : SUCCESS in 25s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/563/ : SUCCESS in 12s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/2267/ : FAILURE in 1m 22s

**Review by no23reason — APPROVED** [review]

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul-el7/308/ : SUCCESS in 22s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/564/ : SUCCESS in 13s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/2275/ : SUCCESS in 12m 00s

**Pull request merged**